src/lib/babel-config.js

```diff
@@ -4,12 +4,8 @@ export default (env, options={}) => ({
 		[require.resolve('babel-preset-env'), {
 			loose: true,
 			modules: options.modules || false,
+			browsers: options.browsers,
```

Comment:
❤️ options.browsers - babel-config.js will be still usable as a babel preset!

However it should be: `target: { browsers: [...] }` like [here](https://github.com/babel/babel-preset-env#babel-preset-env----) (I've wanted to check what this does - I still don't know, but at least I've noticed it is passed differently 😛 )